server/base/src/main/java/org/apache/accumulo/server/init/Initialize.java

```diff
@@ -143,12 +145,14 @@
   private static final String TABLE_TABLETS_TABLET_DIR = "table_info";
 
   private static LineReader reader = null;
+  private static Terminal terminal = null;
   private static ZooReaderWriter zoo = null;
 
   private static LineReader getLineReader() throws IOException {
-    if (reader == null) {
-      reader = LineReaderBuilder.builder().build();
-    }
+    if (terminal == null)
+      terminal = TerminalBuilder.builder().jansi(false).build();
+    if (reader == null)
+      reader = LineReaderBuilder.builder().terminal(terminal).build();
```

Comment:
I noticed there's a few places in our code, and I think this is one of them, where we don't really need to use a JLine terminal or reader at all. Really, we don't have a need to use it outside of accumulo-shell at all. But, I'm happy to have this change if it fixes the immediate bug. I opened #2203  to address this separately.